soloconnect/.elasticbeanstalk/config.yml

```diff
@@ -21,7 +23,7 @@ global:
   instance_profile: null
   platform_name: null
   platform_version: null
-  profile: null
+  profile: eb-cli
```

Comment:
I think this file should be git ignored - because this is specific to each person's config and it will break locally